src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.Ping.can convert IPv4 addresses to IPv6 with -IPv6 parameter
```powershell
Expected collection @('2001:4860:4860::8888', '2001:4860:4860::8844') to contain $null, but it was not found.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 134
134:             $result.Address.IPAddressToString | Should -BeIn @('2001:4860:4860::8888', '2001:4860:4860::8844')
```